ModelBundle/Repository/NodeRepository.php

```diff
@@ -729,6 +729,25 @@ public function findLastPublished()
 
     /**
      * @param string $nodeType
+     *
+     * @return array
+     */
+    public function findAllCurrentlyPublishedByType($nodeType)
```

Comment:
@alavieille : As discussed on the PR using this, I think this kind of method is dangerous for memory usage and page rendering success